src/neon/azext_neon/aaz/latest/neon/postgres/_create.py

```diff
@@ -0,0 +1,602 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "neon postgres create",
+    is_preview=True,
+)
+class Create(AAZCommand):
+    """Create a Neon Resource
+
+    :example: Organizations_CreateOrUpdate
+        az neon postgres create --resource-group <resource-group-name> --name <neon-resource-name> --location <location> --subscription <subscription-id> --marketplace-details "{subscription-id:<marketplace-subscription-id>,subscription-status:PendingFulfillmentStart,offer-details:{publisher-id:<publisher-id>,offer-id:<offer-id>,plan-id:<plan-id>,plan-name:'Neon Serverless Postgres - Free (Test_Liftr)',term-unit:P1M,term-id:<term-id>}}" --user-details "{first-name:<first-name>,last-name:<last-name>,email-address:<email>,upn:<upn>,phone-number:<phone-number>}" --company-details "{company-name:'<company-name>',country:<country>,business-phone:<business-phone>,office-address:'<office-address>',domain:<domain>,number-of-employees:<number-of-employees>}" --partner-organization-properties "{organization-id:<organization-id>,org-name:'<partner-organization-name>',single-sign-on-properties:{single-sign-on-state:Enable,enterprise-app-id:<enterprise-app-id>,single-sign-on-url:'<sso-url>',aad-domains:['<aad-domain>']}}"
```

Comment:
We usually using fake values in the examples. Please don't use the placeholder mark `<>`. It will cause issues in docs generation.

Reply:
Okay. I will replace it